### ISSUE TITLE
fade(false) fails with "TypeError: element.attr(...) is null"

### ISSUE THREAD

**Issue by ngg** [magjac/d3-graphviz#136]
I'm trying to disable fading, but if I call `fade(false)` the render function fails.
The following example is based on the latest 3.0.1 version, but I had a similar exception with 2.6.1 as well.

Easiest way to reproduce:
Apply the following patch to `examples/basic-add-edge.html`
```diff
diff --git a/examples/basic-add-edge.html b/examples/basic-add-edge.html
index e83677e..783630c 100644
--- a/examples/basic-add-edge.html
+++ b/examples/basic-add-edge.html
@@ -24,6 +24,7 @@ var dotSrc = `strict digraph {
 }`;
 
 var graphviz = d3.select("#graph").graphviz()
+    .fade(false)
     .attributer(attributer)
     .transition(function() {
         return d3.transition().duration(1000);
```

The following is shown in the log:
```
TypeError: element.attr(...) is null element.js:36:29
    extractElementData element.js:36
    attributeElement render.js:171
    selection_each d3.js:1176
    insertChildren render.js:103
    attributeElement render.js:329
    selection_each d3.js:1176
    insertChildren render.js:103
    attributeElement render.js:329
    _render render.js:386
    render render.js:24
    <anonymous> self-hosted:876
    1 d3.js:3232
    timerFlush d3.js:3176
    wake d3.js:3186
```

**Comment by magjac** [comment]
Thanks for an excellent bug report. I can confirm I get the same behavior.

**Comment by magjac** [comment]
I'm currently investigating the root cause, but if you want a quick workaround you can add `.tweenShapes(false)`.

**Comment by magjac** [comment]
BTW, the example is broken for other reasons even if using the workaround. I will push an updated example shortly.

**Comment by magjac** [comment]
Pushed a fixed example in https://github.com/magjac/d3-graphviz/commit/5b7a87a4ec5ea65f89aac565a5f0788875f10deb.

Still investigating the bug.

**Comment by ngg** [comment]
I originally had the bug with a really different program, I just wanted to provide a quick reproduction case, so I tried to set fade(false) with the first example I found, and the bug occurred there as well.

**Comment by magjac** [comment]
Thanks. Can you confirm that the `.tweenShapes(false)` workaround avoids the problem in your application?

**Comment by magjac** [comment]
Fixed in [v3.0.2](https://github.com/magjac/d3-graphviz/blob/master/CHANGELOG.md#302)

**Comment by ngg** [comment]
Thanks for the quick fix, it solves the original occurrence of this bug as well.
